src/environment.js

```diff
@@ -135,6 +135,13 @@ export default class Environment extends EventEmitter {
       this.dir = process.cwd();
     }
 
+    if (!this.dest) {
+      this.dest = 'sassdoc';
+    }
+
+    this.dest = this.resolve(this.dest, this.destCli);
+    this.destName = path.relative(process.cwd(), this.dest);
```

Comment:
Then yeah, either `displayDest` and `displayTheme`.
Or maybe a method `display()` which could be used like `env.display('themeName')` ?